packages/shared/src/utils/pdf/languageContext.jsx

```diff
@@ -16,13 +16,13 @@ export const useLanguageContext = () => {
 
 export const withLanguageContext = Component => props => {
   const context = useLanguageContext();
-  const { translations, settings, ...other } = props;
```

Comment:
need to leave settings in props so that they make it in to the pdf component further down